### PR TITLE
feature/ECCMID-1828_Adjust_LongText

### PR DESCRIPTION
**Pull request by williamricardoardila**

### CODE REVIEW

**Review by RonaldRuizCondor — CHANGES_REQUESTED** [review]

File: EmeraldIOS/Components/SwiftUi/TextView/EmeraldTextViewUI/EmeraldNormalTextView/1.0/EmeraldNormalTextView.swift [hunk 1]
```diff
@@ -69,17 +69,30 @@ public struct EmeraldNormalTextView: View {
...
-            .overlay(RoundedRectangle(cornerRadius: Constants.EmeraldSwiftUiTextField.cornerRadius)
-                        .stroke(EmeraldSwiftUiTextView.getBorderColor(
-                                    errorText: errorText,
-                                    focused: focused,
-                                    disabled: disabled),
-                                lineWidth: EmeraldSwiftUiTextView.getBorderWidth(focused: focused)))
+           
     }
     
+     func validatePositionText() -> CGFloat {
```
[the remark this answers is not in the document]
Comment (williamricardoardila) [reply EmeraldIOS/Components/SwiftUi/TextView/EmeraldTextViewUI/EmeraldNormalTextView/1.0/EmeraldNormalTextView.swift:78]: done

File: EmeraldIOS/Components/SwiftUi/TextView/EmeraldTextViewUI/EmeraldNormalTextView/1.0/EmeraldNormalTextView.swift [hunk 2]
```diff
@@ -69,17 +69,30 @@ public struct EmeraldNormalTextView: View {
...
+         if label.isEmpty {
+             return -Constants.EmeraldSwiftUiTextField.topContentSpacing
+         } else  {
+             if !text.isEmpty {
+                 return -(Constants.EmeraldSwiftUiTextField.topContentSpacing - Constants.EmeraldSwiftUiTextField.spaceBetweenContent)
+
```
[the remark this answers is not in the document]
Comment (williamricardoardila) [reply EmeraldIOS/Components/SwiftUi/TextView/EmeraldTextViewUI/EmeraldNormalTextView/1.0/EmeraldNormalTextView.swift:84]: done

File: EmeraldIOS/Components/SwiftUi/TextView/EmeraldTextViewUI/EmeraldNormalTextView/1.0/EmeraldNormalTextView.swift [hunk 3]
```diff
@@ -122,8 +134,8 @@ public struct EmeraldNormalTextView: View {
         .offset(
             x: .zero,
             y: focused || !$text.wrappedValue.isEmpty ?
-            geo.size.height - 100 :
-                geo.size.height - 90  )
+            geo.size.height - 115 :
```
[the remark this answers is not in the document]
Comment (williamricardoardila) [reply EmeraldIOS/Components/SwiftUi/TextView/EmeraldTextViewUI/EmeraldNormalTextView/1.0/EmeraldNormalTextView.swift:137]: done

File: EmeraldIOS/Components/SwiftUi/TextView/EmeraldTextViewUI/EmeraldNormalTextView/1.0/EmeraldNormalTextView.swift [hunk 4]
```diff
@@ -69,17 +69,29 @@ public struct EmeraldNormalTextView: View {
...
                 .bottom, Constants.EmeraldSwiftUiTextField.spaceBottomContent)
-            .overlay(RoundedRectangle(cornerRadius: Constants.EmeraldSwiftUiTextField.cornerRadius)
-                        .stroke(EmeraldSwiftUiTextView.getBorderColor(
-                                    errorText: errorText,
-                                    focused: focused,
-                                    disabled: disabled),
-                                lineWidth: EmeraldSwiftUiTextView.getBorderWidth(focused: focused)))
     }
     
+     func validatePositionText() -> CGFloat {
```
Comment (RonaldRuizCondor) [inline EmeraldIOS/Components/SwiftUi/TextView/EmeraldTextViewUI/EmeraldNormalTextView/1.0/EmeraldNormalTextView.swift:77]: The method must have an order, move the validation to bottom of the class or an extension

**Review by RonaldRuizCondor — CHANGES_REQUESTED** [review]

File: EmeraldIOS/Components/SwiftUi/TextView/EmeraldTextViewUI/EmeraldNormalTextView/1.0/EmeraldNormalTextView.swift (hunk 4 above)
Comment (williamricardoardila) [reply EmeraldIOS/Components/SwiftUi/TextView/EmeraldTextViewUI/EmeraldNormalTextView/1.0/EmeraldNormalTextView.swift]: done

**Review by RonaldRuizCondor — APPROVED** [review]

**Review by lborrego037 — APPROVED** [review]

**Review by lborrego037 — APPROVED** [review]

**Review by lborrego037 — APPROVED** [review]

**Review by GioBCondor — APPROVED** [review]

**Pull request merged**